templates/search/index.html.tera

```diff
@@ -0,0 +1,47 @@
+{% extends "base" %}
+{% import "macros" as macros %}
+
+{% block title %}
+Search
+{% endblock title %}
+
+{% block head %}
+<script>
+window.onload = function(evt) {
+	var form = document.getElementById('form');
+	form.addEventListener('submit', function () {
+	    for (var input of form.getElementsByTagName('input')) {
+		if (input.name && !input.value) {
```

Comment:
You can add this condition before to fix your issue:

```js
if (input.name === '') {
    input.name = input.id
}
```

Reply:
I'll do it then